smart-contracts-tutorial/first-contract.md

```diff
@@ -109,30 +101,53 @@ make scenario3-run-client
 
 Open a web browser to http://localhost:8000/
 
-<small>
+```
 currently, this is the UI of the Pixel Gallery demo
 TODO : provide a proper UI
-</small>
+```
 
 In the REPL, try the following commands in the same order:
 ```
 home
-home!game!guess(14)
-home!game!guess(37)
+host = home~.contractHost
+installation = host~.install({start: home.guess37ContractSource})
+invites = installation~.spawn()
+playerInvite = invites~.playerInvite
+seat = host~.redeem(playerInvite)
+
+seat~.guess(14)
+seat~.guess(37)
 ```
 
-The first `home` command shows what was defined in `lib/ag-solo/vats/vat-demo.js` as `chainBundle`
+### Commands breakdown 
+
+The first `home` command shows what was defined in `lib/ag-solo/vats/vat-pixel.js` as return value of `createPixelBundle`
+It is a presence for an object containing the properties `contractHost`, `handoffService` and `guess37ContractSource`
```

Comment:
Home is an object with fields in the local vat. The objects in the fields are all promises or presences, but home is local.